.github/workflows/update-image-tag.yml

```diff
@@ -0,0 +1,76 @@
+name: Update image tag
+
+on:
+  repository_dispatch:
+    types: [update-image-tag]
+  workflow_dispatch:
+    inputs:
+      image_tag:
+        description: 'Image tag to deploy'
+        required: true
+        type: string
+      repo:
+        description: 'The application repository'
+        required: true
+        type: string
+        default: 'collections'
```

Comment:
Do we want a default here? Or is this just for testing for now?

Reply:
Yeah this is just a demo config for testing.